packages/lib/modules/hooks/useHook.ts

```diff
@@ -0,0 +1,13 @@
+import { Address } from 'viem'
+import { Pool } from '../pool/PoolProvider'
+
+export function useHook(pool: Pool) {
+  const hasHook = !!pool.hook?.address
```

Comment:
Out of curiosity: cannot we have more than one hook in a pool?

Reply:
Nope.